acceptance/setup/git/pre-suite/010_TestSetup.rb

```diff
@@ -38,7 +38,26 @@
 END
         case host['platform']
         when /windows/
-          raise ArgumentError, "Windows is not yet supported"
+          create_remote_file(host, "#{puppet_dir}/Gemfile", gemfile_contents)
+          # bundle must be passed a Windows style path for a binstubs location
+          binstubs_dir = on(host, "cygpath -m \"#{host['puppetbindir']}\"").stdout.chomp
+          # note passing --shebang to bundle is not useful because Cygwin
+          # already finds the Ruby interpreter OK with the standard shebang of:
+          # !/usr/bin/env ruby
+          # the problem is that a Cygwin style path is passed to the interpreter
+          on host, "cd #{puppet_dir} && cmd.exe /c \"bundle install --system\ --binstubs #{binstubs_dir}\""
```

Comment:
sorry, i guess my question could also have been worded: "why is the space escaped?"